src/main/java/io/nats/client/Options.java

```diff
@@ -457,6 +457,11 @@ public class Options {
      * to do TLS upgrade first, before INFO
      */
     public static final String PROP_TLS_FIRST = PFX + "tls.first";
+    /**
+     * Property used to configure to skip checking whether the client and server
+     * match that they should both be secure or both be insecure.

```

Comment:
I would explain the use case in the comment.
It's pretty NATS specific :).

Reply:
Updated.